core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -55,7 +55,7 @@
             TESTCONTAINERS_SESSION_ID_LABEL, SESSION_ID
     );
 
-    private static final String TINY_IMAGE = TestcontainersConfiguration.getInstance().getTinyImage();
+    private static final String TINY_IMAGE = TestcontainersConfiguration.getInstance().getTinyDockerImageName().toString();
```

Comment:
I always get a strange feeling when `toString()` is used outside of a context of e.g. logging or being a human-readable presentation.

Reply:
Yeah, TBH I had a slight feeling of discomfort writing it. I think I’ll change it.

Reply:
Oh, the new one is much better! I struggled to find a good method name, but I think `canonicalName` works pretty good :)